src/main/java/io/cdap/e2e/pages/locators/CdfNameSpaceAdminLocators.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.e2e.pages.locators;
+
+import io.cdap.e2e.utils.SeleniumDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+/**
+ * Represents CdfNameSpaceAdminLocators
+ */
+public class CdfNameSpaceAdminLocators {
+
+  @FindBy(how = How.XPATH, using = "//div[contains(text(),'Namespace Admin')]")
+  public static WebElement namespaceAdminMenu;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='navbar-hamburger-icon']")
+  public static WebElement hamburgerMenu;
+
+  public static WebElement nameSpaceModules(String module, String nameSpace) {
+    String path = "//*[@href=\"/cdap/ns/" + nameSpace + "/details/" + module + "\"]";
+    return SeleniumDriver.getDriver().findElement(By.xpath(path));
+  }
+
+  public static WebElement createProfile(String nameSpace) {
+    String path = "//*[@href=\"/cdap/ns/" + nameSpace + "/profiles/create\"]";
+    return SeleniumDriver.getDriver().findElement(By.xpath(path));
+  }
+
+  public static WebElement switchNameSpace(String nameSpace) {
+    String path = "//*[@href=\"/cdap/ns/'" + nameSpace + "'\"]";
+    return SeleniumDriver.getDriver().findElement(By.xpath(path));
+  }
+
+  public static WebElement locateMenuLink(String menuLink) {
+    String xpath = "//*[@data-cy='navbar-" + menuLink + "-link']";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  public static WebElement locateKeyProperty(String pluginProperty, int row) {
+    String xpath = "//*[@data-cy='" + pluginProperty + "" + row + "']//input[@placeholder='key']";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  public static WebElement locateValueProperty(String pluginProperty, int row) {
+    String xpath = "//*[@data-cy='" + pluginProperty + "" + row + "']//input[@placeholder='value']";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  public static WebElement locateAddRowButtonProperty(String pluginProperty, int row) {
```

Comment:
Same comment applies to this file.

Reply:
Ok ankit , I have done the necessary changes . Please have a look in the latest commit . Thanks .